src/parsing/test/testing.ts

```diff
@@ -348,13 +299,30 @@ export function toEqualDefinition(actualResponse, testDefinition) {
         pass:
             !testDefinition.expectedResult ||
             resultEquals(actualResponse, testDefinition.expectedResult),
-        message: () =>
-            '-------- Statement: ' + testDefinition.beforeCursor + '|' + testDefinition.afterCursor + '\n' +
-            '-- Expected response: ' + jsonStringToJsString(JSON.stringify(testDefinition.expectedResult)) + '\n' +
-            '-- Parser response: ' + jsonStringToJsString(JSON.stringify(actualResponse)) + '\n'
+        message: constructTestCaseMessage(testDefinition, {
+            'Expected errors': testDefinition.expectedResult,
+            'Parser errors': actualResponse,
+        }),
     };
 }
 
+function constructTestCaseMessage(testCase: TestCase, details: Record<string, any>): () => string {
+    let message = `- Query: ${testCase.beforeCursor + '|' + testCase.afterCursor} \n`;
+
+    for (let key in details) {
+        let value = details[key];
+
+        let strValue = value;
+        if (typeof value != 'string') {
+            strValue = jsonStringToJsString(JSON.stringify(value))
+        }
```

Comment:
I think that `JSON.stringify` should work just fine for any type of data. It can probably be written better, but it's a test function, so we can leave it as is and upgrade if some misfires will be found

Reply:
Seems like strValue is excess variable, lets reduce it maybe

Reply:
How do you want to reduce the code?

Reply:
Let's just mutate value which defined on 313 line

Reply:
Ah, got it. Good, let's do it